doc/release/release_dev.rst

```diff
@@ -98,6 +133,29 @@ Documentation
 - Update reference link documentation in the ``adjust_sigmoid`` function.
 - Fix reference to multiscale_basic_features in TrainableSegmenter.
 - Slight ``shape_index`` docstring modification to specify 2D array.
+- Add stitching gallery example (gh-5365)
+- Add draft SKIP3: transition to scikit-image 1.0 (gh-5475)
+- Mention commit messages in the contribution guidelines. (gh-5504)
+- Fix and standardize docstrings for blob detection functions (gh-5547)
+- Updated the User Guide to reflect usage of ``channel_axis`` rather than
```

Comment:
```suggestion
- Fix and standardize docstrings for blob detection functions. (gh-5547)
- Update the User Guide to reflect usage of ``channel_axis`` rather than
```